### ISSUE TITLE
"Shuffle by Album" can prevent the correct behave of "Repeat Playlist" (and maybe others option)

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#261]
_From [kadaimx](https://code.google.com/u/kadaimx/) on April 30, 2010 01:05:52_

What steps will reproduce the problem? 1. Turn on "Repeat Playlist" and "Shuffle by Album"
2. Load a playlist consisting of 3 of more albums.
3. Start playing a track at random. What is the expected output? What do you see instead? Expected that, once a album is finished, clementine started to play the
next album, but instead, it behaved as having set "Repeat Album". What version of the product are you using? On what operating system? Revision 801 on Kubuntu 10.04 Please provide any additional information below.

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=261_

**Comment by JulianVolodia** [comment]
Hi @kadaimx have you moved from Google Code to GitHub? Could you reproduce issue and if not appear there - could you close it or leave feedback here? Thanks!

Hi @jerobarraco do you think this is duplicate of error you have reported or is it false-positive relation?
